cluster-up/hack/config-default.sh

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
There's a handful of files that have only whitespace changes once the dust has settled. It might be helpful to strip those changes for clarity.